src/Main/Main.js

```diff
@@ -9,6 +11,11 @@ export default function Main() {
       {/* get the zodiac array from props */}
       {/* for each item in the `zodiacs` array props, render a ZodiacCard component, passing in the name and the dates as props to the ZodiacCard component */}
       {/* remember that when you map over an array and render out a list, each item in the list needs a unique 'key' prop */}
+      {zodiac.map((zodiac) => (
```

Comment:
Here you would `props.zodiac.map` instead of mapping the imported array directly.

Reply:
Thank you so much for the detailed feedback I really appreciate it!